maven-core/src/test/java/org/apache/maven/internal/transformation/ConsumerPomArtifactTransformerTest.java

```diff
@@ -81,5 +81,10 @@ public Model getRawModel(String groupId, String artifactId) throws IllegalStateE
         public Model getRawModel(Path p) {
             return null;
         }
+
+        @Override
+        public Path locate(Path path) {
+            return null;
```

Comment:
If this is just a stub in a test that isn't called, I'd prefer `throw new UnsupportedOperationException()`